.meteor/packages

```diff
@@ -170,3 +170,5 @@ yasaricli:slugify
 yasinuslu:blaze-meta
 deepwell:bootstrap-datepicker2
 rocketchat:google-natural-language
+rocketchat:drupal
+rocketchat:i18n
```

Comment:
Was it necessary to add this?  I think our other packages might just add this as a dependency.


@RocketChat/core is there a reason it's not in the packages list?